force-app/testRecordGenerator/staticresources/standard_test_record_generators_json.json

```diff
@@ -16,6 +16,62 @@
       }
     ]
   },
+  "Asset": {
+    "generator": {
+      "MasterLabel": "Asset",
+      "DeveloperName": "Asset",
+      "nebc__Apex_Class__c": "nebc.TestMetadataRecordGenerator",
+      "nebc__Priority__c": 0,
+      "nebc__SObject__c": "Asset"
+    },
+    "fields": [
+      {
+        "MasterLabel": "Asset: Name",
+        "DeveloperName": "Asset_LastName",
```

Comment:
Should the name here be Asset_Name instead of Asset_Lastname?

Reply:
Yes that maybe a little copy and paste issue there :)